Ensures that the LIBFFI_INCLUDEDIR environment variable can be forced

*** configure.ac.orig	2020-04-13 14:50:17.476803111 +0000
--- configure.ac	2020-04-13 14:52:29.859184113 +0000
***************
*** 3011,3022 ****
      with_system_ffi="yes"
  fi
  
! if test "$with_system_ffi" = "yes" && test -n "$PKG_CONFIG"; then
!     LIBFFI_INCLUDEDIR="`"$PKG_CONFIG" libffi --cflags-only-I 2>/dev/null | sed -e 's/^-I//;s/ *$//'`"
  else
!     LIBFFI_INCLUDEDIR=""
  fi
- AC_SUBST(LIBFFI_INCLUDEDIR)
  
  # Check for use of the system libmpdec library
  AC_MSG_CHECKING(for --with-system-libmpdec)
--- 3011,3028 ----
      with_system_ffi="yes"
  fi
  
! if test -z "${LIBFFI_INCLUDEDIR+x}"
! then
!     if test "$with_system_ffi" = "yes" && test -n "$PKG_CONFIG"; then
!         LIBFFI_INCLUDEDIR="`"$PKG_CONFIG" libffi --cflags-only-I 2>/dev/null | sed -e 's/^-I//;s/ *$//'`"
!     else
!         LIBFFI_INCLUDEDIR=""
!     fi
!     AC_SUBST(LIBFFI_INCLUDEDIR)
  else
!     AC_SUBST(LIBFFI_INCLUDEDIR, "${LIBFFI_INCLUDEDIR}")
!     AC_MSG_NOTICE([Using LIBFFI_INCLUDEDIR=${LIBFFI_INCLUDEDIR}])
  fi
  
  # Check for use of the system libmpdec library
  AC_MSG_CHECKING(for --with-system-libmpdec)
